### ISSUE TITLE
[DevTools Docs Feedback] Incorrect documentation of `CoreWebView2.AddHostObjectToScript` for WinRT

### ISSUE THREAD

**Issue by softworkz** [MicrosoftDocs/edge-developer#2837]
The documentation for AddHostObjectToScript is incorrect. It's obviously copied from the documentation for the .NET version of WebView2, but for WinRT, the procedures are different (and complicated unfortunately): 

https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.2065-prerelease#addhostobjecttoscript

For WinRT, an insane effort is required which involves creating a separate project for the classes that should be exposed and it also requires adding a C++ project (like WinRtAdapter in the code sample), which creates WinRT/COM definitions, which finally need to be wrapped again via methods from ICoreWebView2DispatchAdapter.
(roughly like that)

These procedures are not documented and the current description is wrong for WinRT.

**Comment by mikehoffms** [comment]
Related article:
Call native-side WinRT code from web-side code
https://learn.microsoft.com/microsoft-edge/webview2/how-to/winrt-from-js

**Comment by softworkz** [comment]
> Related article: Call native-side WinRT code from web-side code [learn.microsoft.com/microsoft-edge/webview2/how-to/winrt-from-js](https://learn.microsoft.com/microsoft-edge/webview2/how-to/winrt-from-js)

Yes, I know that article - which is providing correct information.

But the part I referenced above is incorrect.

**Comment by captainbrosset** [comment]
Thank you for reporting this. I'm making sure this goes into our backlog.

**Comment by mikehoffms** [comment]
added 2nd AB# at top, for Bug that's being triaged
